README.md

```diff
@@ -42,90 +48,144 @@ pip install ctgan
+Once you have the data ready, you need to import and create an instance of the `CTGANSynthesizer`
+class and fit it passing your data and the list of discrete columns.
 
-```
-C    0    500
-D    A    B    C
-D    True     False
+```python
+from ctgan import CTGANSynthesizer
+
+ctgan = CTGANSynthesizer()
+ctgan.fit(data, discrete_columns)
 ```
 
-## Run model
+This process is likely to take a long time to run.
+If you want to make the process shorter, or longer, you can control the number of training epochs
+that the model will be performing by adding it to the `fit` call:
 
-```
-USAGE:
-    python3 ctgan/cli.py [flags]
-flags:
-  --data: Filename of training data.
-    (default: '')
-  --max_epoch: Epoches to train.
-    (default: '100')
-    (an integer)
-  --meta: Filename of meta data.
-    (default: '')
-  --model_dir: Path to save model.
-    (default: '')
-  --output: Output filename.
-    (default: '')
-  --sample: Number of rows to generate.
-    (default: '1000')
-    (an integer)
+```python
+ctgan.fit(data, discrete_columns, epochs=5)
 ```
 
-## Example
+## 2. Generate synthetic data
 
-It's easy to try our model using example datasets.
+Once the process has finished, all you need to do is call the `sample` method of your
+`CTGANSynthesizer` instance indicating the number of rows that you want to generate.
 
+```python
+samples = ctgan.sample(1000)
 ```
-git clone https://github.com/DAI-Lab/ctgan
-cd ctgan
-python3 -m ctgan.cli --data examples/adult.dat --meta examples/adult.meta
-```
-
-
-## What's next?
 
-For more details about **CTGAN** and all its possibilities
-and features, please check the [documentation site](https://DAI-Lab.github.io/CTGAN/).
+The output will be a table with the exact same format as the input and filled with the synthetic
+data generated by the model.
+
+|     age | workclass    |    fnlwgt | ... |   hours-per-week | native-country   | income   |
+|---------|--------------|-----------|-----|------------------|------------------|----------|
+| 26.3191 | Private      | 124079    | ... |          40.1557 | United-States    | <=50K    |
+| 39.8558 | Private      | 133996    | ... |          40.2507 | United-States    | <=50K    |
+| 38.2477 | Self-emp-inc | 135955    | ... |          40.1124 | Ecuador          | <=50K    |
+| 29.6468 | Private      |   3331.86 | ... |          27.012  | United-States    | <=50K    |
+| 20.9853 | Private      | 120637    | ... |          40.0238 | United-States    | <=50K    |
+|     ... | ...          |       ... | ... |              ... | ...              | ...      |
+
+
+**NOTE**: CTGAN does not distinguish between float and integer columns, which means that it will
+sample float values in all cases. If integer values are required, the outputted float values
+must be rounded to integers in a later step, outside of CTGAN.
+
```

Comment:
I think we should provide some evidence to show the performance of CTGAN. For example, the prediction accuracy of CTGAN on demo model.

Reply:
I think it would be better do this on a later PR, to keep avoid making this one bigger.